src/local/CMakeLists.txt

```diff
@@ -0,0 +1,30 @@
+# add include directory to compiler switches
+include_directories(${STIR_LOCAL}/include)
```

Comment:
I don't think we should have this line. That's the business of the extension

Reply:
Sorry, I don't understand. That's the business of which extension?